openapi.yaml

```diff
@@ -13383,12 +13690,14 @@ components:
       format: date-time
       example: '2022-11-04T20:10:06.927Z'
       readOnly: true
+      x-speakeasy-param-suppress-computed-diff: true
```

Comment:
Change to `CreatedAt` and `UpdatedAt` are intended? They seem to lead to changes in files related to API Product, etc in `/internal` directory
Also, I see we have introduced new schemas `schemas-CreatedAt` and `schemas-UpdatedAt` - any difference between these and `CreatedAt` and `UpdatedAt`?

Reply:
Let me regenerate this. I simplified the schemas yesterday as they’re identical